LayoutTests/http/wpt/service-workers/coreipc.js

```diff
@@ -0,0 +1,1244 @@
+const MAX_UINT8 = Math.pow(2, 8)-1;
```

Comment:
Rather than making a copy, can we add an alias to `/ipc/coreipc.js` in `/Tools/Scripts/webkitpy/layout_tests/servers/aliases.json`?